### PR TITLE
fix: achievement release changes

### PR DESCRIPTION
**Pull request by dhenkel92**
Some small adaption for things that we noticed during our last testing session

### CODE REVIEW

File: common/achievement/metric.ts [hunk 1]
```diff
@@ -40,9 +40,13 @@ const batchOfMetrics = [
     createMetric('pupil_onboarding_appointment_booked', ['pupil_calendly_appointment_booked'], () => {
         return 1;
     }),
-    createMetric('pupil_onboarding_screened', ['pupil_screening_appointment_done', 'pupil_screening_add', 'pupil_screening_succeeded'], () => {
-        return 1;
-    }),
+    createMetric(
+        'pupil_onboarding_screened',
+        ['pupil_screening_appointment_done', 'pupil_screening_add', 'pupil_screening_after_registration_succeeded'],
```
Comment (dhenkel92) [inline common/achievement/metric.ts:45]: The action was changed recently.

File: common/achievement/index.ts [hunk 2]
```diff
@@ -168,6 +169,8 @@ async function rewardUser<ID extends ActionID>(evaluationResult: number | null,
 
     const { type, group, groupOrder } = updatedAchievement.template;
 
+    // We want to delay the notification by one minute
+    const triggerNotificationAt = moment().add(1, 'minute').toDate();
```
Comment (dhenkel92) [inline common/achievement/index.ts:173]: @Jonasdoubleyou Does this work with user-app notifications?

Comment (dhenkel92) [reply common/achievement/index.ts:173]: It seems like it will be delivered during the next `checkReminders` schedule, which is every ten minutes...

Comment (Jonasdoubleyou) [reply common/achievement/index.ts]: exactly

**Pull request merged**